### PR TITLE
Firebase changes

### PR DESCRIPTION
**Pull request by paglias**

### CODE REVIEW

File: test/api/groups.coffee [hunk 1]
```diff
@@ -151,7 +151,7 @@ describe "Guilds", ->
           done()
 
   context "removing users groups", ->
-    it "allows guild leaders to remove a member", (done) ->
+    it "allows guild leaders to remove a member (but not themselves)", (done) ->
```
Comment (paglias) [inline test/api/groups.coffee:154]: @crookedneighbor I've modified a test to be sure a change is working, it's ok to modify it or you prefer a new test to be created?

Comment (crookedneighbor) [inline test/api/groups.coffee:154]: Technically, each test should be testing one thing, ie: `it "allows guild leaders to remove a member"`, `it "does not allow guild leader to remove themselves"`, `it "does not allow non-guild leaders to remove a member"`

Though, that said, we're deprecating these tests soon in favor of more app-agnostic api tests. I'm hoping to set up the new testing suite soon (in ES2015!), so in the meantime, do whatever you want with these

**Comment by paglias** [comment]
@crookedneighbor I plan to do some other changes, I'll add a "WIP: do not merge" at the top

**Comment by paglias** [comment]
in the next PR

**Comment by crookedneighbor** [comment]
Makes sense. This one seemed more like a minor bug fix than anything related specifically to firebase.

**Comment by paglias** [comment]
Yes, I just started looking at the code, found this small bug and decided to fix it but it's not specifically related

**Pull request merged**